src/metabase/driver/sql_jdbc/actions.clj

```diff
@@ -205,12 +205,11 @@
 
 (defmethod actions/perform-action!* [:sql-jdbc :row/delete]
   [driver action database {database-id :database, :as query}]
-  (let [raw-hsql    (mbql-query->raw-hsql driver query)
-        delete-hsql (-> raw-hsql
-                        (dissoc :select :limit)
-                        (assoc :delete [])
```

Comment:
What did this `delete` key do before? 🫤

Reply:
not even that, that's for deleting from multiple tables!? Fancy!